src/pages/components/lead-space-block.mdx

```diff
@@ -13,7 +13,9 @@ import ResourceLinks from 'components/ResourceLinks';
 
 <AnchorLink>Resources</AnchorLink>
```

Comment:
@RichKummer small comment: need to move the Resources links between Modifiers & Content guidance here in the anchor links and in the body of the page😄